### PR TITLE
Flake module

### PR DESCRIPTION
**Pull request by roberth**
Done.

>   A module to import into flakes based on flake-parts.
>  Makes integration into a flake easy and tidy.
>  See https://github.com/hercules-ci/flake-parts#readme

### CODE REVIEW

**Comment by domenkozar** [comment]
Hey Robert, do you plan to work on this one further?

**Comment by roberth** [comment]
It's actually in a good state since the latest iteration. The remaining items weren't at all necessary and I've already incorporated feedback from Jonas.

**Comment by roberth** [comment]
Rebased, ready and green :)

File: .envrc [hunk 1]
```diff
@@ -1,4 +1,6 @@
+HOST_XDG_DATA_DIRS="${XDG_DATA_DIRS:-}"
```
Comment (domenkozar) [inline .envrc:1]: I'd really like to get rid of lorri :( I think it's overengineered and with flakes not really needed.

**Pull request merged**